acceptance/extension-logging-fluentd/package.json

```
@@ -6,7 +6,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "engines": {
-    "node": ">=10.16"
+    "node": "^10.16 || ^12 || ^14"
```

Do we anticipate any breaking changes when Node.js 15.0 is released? If not, then I prefer to use `>=14` to accommodate future major versions without having to update all `package.json` files again.

Also: I think `^12` can be abbreviated as just `12`.

```
$ npx semver -r 12 11.0.0 12.0.0 12.11.10 13.0.0
12.0.0
12.11.10
```

Proposed value to use:

```suggestion
    "node": "^10.16 || 12 || >=14"
```

Thoughts?

```
$  npx semver -r "^10.16 || 12 || >=14" 10.1.0 10.16.0 11.1.0 12.1.0 13.1.0 14.1.0 15.1.0
10.16.0
12.1.0
14.1.0
15.1.0
```

---

> Do we anticipate any breaking changes when Node.js 15.0 is released?

Not that I know of, but I think it's preferable to use `engines` as the "officially-supported Node.js versions" indicator, while anything outside of that range being "soft supported", but not tested by the CI.